ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_additional.robot

```diff
@@ -17,19 +18,12 @@ ${PRJ_RESOURCE_NAME}=   ods-ci-ds-project-test-additional
 ${PRJ_DESCRIPTION}=   ${PRJ_TITLE} is a test project for validating DS Project feature
 ${TOLERATIONS}=    workbench-tolerations
 ${TOLERATIONS_2}=    workbench-tolerations-two
-${DEFAULT_TOLERATIONS}=    NotebooksOnly   
+${DEFAULT_TOLERATIONS}=    NotebooksOnly
 ${WORKBENCH_TITLE_TOL_1}=   ODS-CI Workbench Tol
 ${WORKBENCH_TITLE_TOL_2}=   ODS-CI Workbench Tol 2
 ${WORKBENCH_DESCRIPTION}=   a test workbench to check tolerations are applied
 ${WORKBENCH_TITLE_GPU}=   ODS-CI Workbench GPU
 ${WORKBENCH_DESCRIPTION_GPU}=   ${WORKBENCH_TITLE_GPU} is a test workbench using GPU
-${NB_IMAGE}=        Minimal Python
-${NB_IMAGE_GPU}=        PyTorch
-${PV_NAME_TOL_1}=         ods-ci-tolerations
-${PV_NAME_TOL_2}=         ods-ci-tolerations-2
-${PV_NAME_GPU}=         ods-ci-gpu
-${PV_DESCRIPTION}=         it is a PV created to test DS Projects feature
-${PV_SIZE}=         1
```

Comment:
these should remain here because are test values

Reply:
Done